darwin/future/meta/objects/dataset.py

```diff
@@ -162,9 +155,3 @@ def _validate_slug(slug: str) -> None:
 
         VALID_SLUG_CHARS = "abcdefghijklmnopqrstuvwxyz0123456789-_"
         assert_is(all(c in VALID_SLUG_CHARS for c in slug_copy), "slug must only contain valid characters")
-
```

Comment:
It's nice doing a review that is largely deleting code that has been made unnecessary by simplification. 👍🏻